Dashboard/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <!-- Required meta tags -->
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    
+        <!-- Bootstrap CSS -->
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    
+        <title>REMi dashboard</title>
+        <link rel="stylesheet" href="main.css"
+      </head>
+      <body>
+        <div class="container">
+          <!-- Navigation Bar -->
+          <nav class="navbar navbar-expand-sm navbar-dark bg-dark">
+            <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse">
+              <span class="navbar-toggler-icon"></span>  
+            </button>
+            <a class="navbar-brand" href="#">REMi</a>
+            <div class="collapse navbar-collapse" id="navbarNav">
+              <ul class="navbar-nav">
+                <li class="nav-item active">
+                  <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>  
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#">How To</a>
+                </li>  
+                <li class="nav-item">
+                  <a class="nav-link" href="#">Leaderboard</a>  
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#">Repository</a>  
+                </li>
+              </ul>  
+            </div>
+          </nav>
+        </div>
+        <h1 style="text-align:center;">Welcome to REMi</h1>
+    
+        <!-- Optional JavaScript -->
+        <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+        <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+      </body>
+</html>
```

Comment:
Not a project-breaking issue, but good practice since some machines are very strict with having an extra line at the end of every file (as this Github warning suggested). 

Fix this in both files and it'll be ready to merge!